requirements/specifications/hardware/hdmi-input.md

```diff
@@ -286,3 +286,28 @@ HDMIInput.autoLowLatencyModeCapableChanged((data) => {
 `port` - the HDMI port that had an E-EDID ALLM advertisement change.
 
 The `onAutoLowLatencyModeCapableChanged`  API requires `use` access to the `xrn:firebolt:capability:inputs:hdmi` capability.
+
+
+## 9. HDMI Port Selection
+
+The `HDMIInput` module **MUST** have a `select` method that manages the selection of on a single HDMI port for playback.
+
+The `HDMIInput.select` **MUST** include a `port` field complying to the schema of `HDMIPortId`
+
+The `HDMIInput.select` **MUST** include a `type` field complying to the schema of `HDMISelectType`
```

Comment:
Why not make this two methods: `start(port)` and `stop(port)`?

We should check what the HDMI specification itself calls this? e.g. is it start/activate, etc., and use the same language.

Reply:
Based on the discussion below changes were implemented
1. Change `select` to `open` and `close` which are more standard to HDMI terminology.
2. Make the changes in such a way that they are consistent and reusable with the Composite input support.